build.gradle

```diff
@@ -1,9 +1,11 @@
 buildscript {
     repositories {
-        jcenter()
+        maven {
```

Comment:
@vlsi is the maven section really required? Its listed on the plugin page [com.github.johnrengelman.shadow](https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow but it was not required for the 1.2.4 version